client/src/utils/botExecution.jsx

```diff
@@ -0,0 +1,8 @@
+// is a dummy function
+const isBotExecutable = () => {
+    if (Math.random() > 0.49) {
+        return true;
+    }
+    return false;
+};
+export default isBotExecutable;
```

Comment:
One small thing: In itself a great idea to include something like math.random, however it confused me a lot when testing it. With some clicks I got an alert window, with other clicks not. I thought the app was not responding. Maybe next time you might just return a constant alert with "... is not implemented yet"

Reply:
Good point. I haven't thought of it myself but I agree with Lukas :)

Reply:
The isBotExecutable function is to simply a dummy function that is to be implemented in this sprint. As I would want the error handling functionality to be shown (is an acceptance criterion) I wanted it to be displayed. But I also wanted to show, that the current functionality is still there. Therefore, I do the error handling with a random probability to spice things up.